keys.h

```diff
@@ -105,7 +105,8 @@ enum key_cmd
 	KEY_CMD_LYRICS,
 	KEY_CMD_QUEUE_TOGGLE_FILE,
 	KEY_CMD_QUEUE_CLEAR,
-	KEY_CMD_WRONG
+	KEY_CMD_WRONG,
+	KEY_CMD_DELETE_CURRENT_FILE
```

Comment:
Don't you have to switch the order, so that KEY_CMD_WRONG is always the last entry?

Reply:
I went through the source code, I don't see anywhere the order of the enum considered for anything. If there is a reason, I'm happy to change it.